### PR TITLE
README: Add project context

### LINKED ISSUE #38: Readme overhaul
(see https://github.com/lagom/online-auction-scala/issues/24)

The project's Readme should list:

 * project context
 * instructions to import into an IDE (see https://github.com/lagom/online-auction-scala/issues/23 and https://github.com/lagom/online-auction-scala/issues/22)

### PR DESCRIPTION
**Pull request by yg-apaza**
Following the example given by @nomthysmylz, I added the project context for Java app.
Fixes #38

### CODE REVIEW

**Comment by yg-apaza** [comment]
:) So, what's next? @rstento suggested to add description on how to use it
lagom/online-auction-scala#25
Should I do that here too?

**Comment by TimMoore** [comment]
@yg-apaza I think we can add that separately. It probably makes sense for one person to write the text that will be used in both the Java & Scala versions, since the functionality is the same (or is intended to be, once they're both complete).

File: README.md [hunk 1]
```diff
@@ -1,23 +1,34 @@
+[<img src="docs/logo.png" alt="Lagom" width="50%">](https://github.com/lagom/lagom)
+
+[![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/lagom/lagom?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [<img src="https://img.shields.io/travis/lagom/lagom.svg"/>](https://travis-ci.org/lagom/lagom)
+
 # INTRO
 
-This is a sample app of the Lagom Framework.
+Lagom is a Swedish word meaning just right, sufficient. Microservices are about creating services that are just the right size, that is, they have just the right level of functionality and isolation to be able to adequately implement a scalable and resilient system.
+
```
Comment (TimMoore) [inline README.md]: Looks like this is missing a paragraph that is in the Scala version: https://github.com/lagom/online-auction-scala/pull/25/files#diff-04c6e90faac2675aa89e2176d2eec7d8R8

File: README.md [hunk 2]
```diff
@@ -45,3 +56,10 @@ The auction system is the sum of 5 micro-services and a web gateway:
 * **web-gateway**: a [Play](https://www.playframework.com/) application providing web UI and acting as gateway to all previously described services.
 
 Check the docs for each service for details on the **commands** and **queries** it serves as well as events the service **emits** and events it **consumes** from the [Message Broker](http://www.lagomframework.com/documentation/1.3.x/java/MessageBroker.html#Message-Broker-Support).
+
+## Good to know
+
+- Hello World Giter8 seed
+For a simple, gentler, introduction to Lagom with Java, have a look at the Hello World giter8 seed project using [Maven](http://www.lagomframework.com/documentation/1.3.x/java/GettingStartedMaven.html) or [sbt](http://www.lagomframework.com/documentation/1.3.x/java/GettingStartedSbt.html).
```
Comment (TimMoore) [inline README.md]: To be totally accurate, the Maven project doesn't use giter8, it is a Maven archetype.

File: README.md [hunk 3]
```diff
@@ -45,3 +58,10 @@ The auction system is the sum of 5 micro-services and a web gateway:
 * **web-gateway**: a [Play](https://www.playframework.com/) application providing web UI and acting as gateway to all previously described services.
 
 Check the docs for each service for details on the **commands** and **queries** it serves as well as events the service **emits** and events it **consumes** from the [Message Broker](http://www.lagomframework.com/documentation/1.3.x/java/MessageBroker.html#Message-Broker-Support).
+
+## Good to know
+
+- Hello World Giter8 seed
```
Comment (ignasi35) [inline README.md]: This title doesn't make sense. In java either sbt's g8 or mvn archetypes could be used.

**Comment by ignasi35** [comment]
Good job @yg-apaza , Thanks!

**Pull request merged**